client/sites-dashboard-v2/sites-dataviews/index.tsx

```diff
@@ -113,13 +113,8 @@ const DotcomSitesDataViews = ( {
 				id: 'status',
 				header: __( 'Status' ),
 				render: ( { item }: { item: SiteInfo } ) => <SiteStatus site={ item } />,
-				type: 'enumeration',
-				elements: siteStatusGroups,
-				filterBy: {
-					operators: [ 'in' ],
-				},
 				enableHiding: false,
-				enableSorting: false,
+				enableSorting: true,
```

Comment:
I think we only want the sorting for the site and the last published column. Hence, we can keep it false, right?